docbot/action.yml

```diff
@@ -8,6 +8,16 @@ branding:
 runs:
   using: composite
   steps:
-    - run: go run main.go
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        repository: pulsar/pulsar-test-infra
+        ref: master
```

Comment:
```suggestion
```

Reply:
The current branch ref is master: https://github.com/apache/pulsar-test-infra/tree/master

Reply:
Fair enough. If we remove this config, the default branch is also master.

Reply:
Removed, let ref uses the default branch.